nodejs/repositories.bzl

```diff
@@ -374,7 +377,7 @@ node_repositories = repository_rule(
 )
 
 # Wrapper macro around everything above, this is the primary API
-def nodejs_register_toolchains(name, register = True, **kwargs):
+def nodejs_register_toolchains(name = DEFAULT_NODE_REPOSITORY, register = True, **kwargs):
```

Comment:
Wdyt of defaulting it for the workspace rule as well?

Reply:
hm, I don't think we need to change that old one, and explicit name is probably better there since there's no transitive semantics

Reply:
Removed